doc/quickstart.md

```diff
@@ -0,0 +1,108 @@
+# vfkit quick start
+
+## Introduction
+
+vfkit is a macOS command-line-based hypervisor, which uses [Apple's Virtualization Framework](https://developer.apple.com/documentation/virtualization?language=objc) to run virtual machines.
+You start a virtual machine by running vfkit with a set of arguments describing the virtual machine configuration/hardware.
+When vfkit stops, the virtual machine stops running.
+It requires macOS 11 or newer, and runs on both x86_64 and aarch64 Macs.
+Some features are only available on macOS 13 or newer.
+
+
+## Installation
+
+You can get vfkit either by downloading it from [its release page](https://github.com/crc-org/vfkit/releases), or get it from [brew](https://brew.sh/):
+```
+# Only the first time
+$ brew tap cfergeau/crc
+
+$ brew install vfkit
+```
+
+
+## Quick start
+
+### Getting a disk image
+
+Your virtual machine will need an operating system to run, so you need to download a disk image first.
+The image needs to be in the raw or iso format. qcow2 or VirtualBox images cannot be used by vfkit.
+
+For example, Fedora images can be downloaded with:
+```
+# For Apple silicon Macs
+$ curl -L -O https://download.fedoraproject.org/pub/fedora/linux/releases/38/Cloud/aarch64/images/Fedora-Cloud-Base-38-1.6.aarch64.raw.xz
+$ xz -d ./Fedora-Cloud-Base-38-1.6.aarch64.raw.xz
+$ mv Fedora-Cloud-Base-38-1.6.aarch64.raw vfkit-test-image.raw
+
+# For Intel Macs
+$ curl -L https://download.fedoraproject.org/pub/fedora/linux/releases/38/Cloud/x86_64/images/Fedora-Cloud-Base-38-1.6.x86_64.raw.xz
+$ xz -d ./Fedora-Cloud-Base-38-1.6.x86_64.raw.xz
+$ mv Fedora-Cloud-Base-38-1.6.x86_64.raw vfkit-test-image.raw
+```
+
+
+### Starting a virtual machine
+
+
+Now that we have a disk image, we can start a virtual machine with 2 virtual CPUs and 2GiB of RAM:
+
+```
+$ vfkit \
+    --cpus 2 --memory 2048 \
+    --bootloader efi,variable-store=efi-variable-store,create \
+    --device virtio-blk,path=vfkit-test-image.raw
+```
+
+No logs from the virtual machine are displayed in the terminal where vfkit was started, but it should show:
+```
+INFO[0000] virtual machine is running
+INFO[0000] waiting for VM to stop
+```
+
+The virtual machine will be running until you hit Ctrl+C or kill the vfkit process.
+If you are using an image which does not support UEFI boot, you can use the [`linux` bootloader](https://github.com/crc-org/vfkit/blob/main/doc/usage.md#linux-bootloader).
+This requires separate kernel, initrd file and kernel command-line arguments.
```

Comment:
I would suggest to use `linux` bootloader as a regular part of sentence and moving the `usage.md` to the end of the following sentence.

`This requires a .... Details about the use can be found in the usage instructions.`